lib/src/utils/string_constants.dart

```diff
@@ -4,6 +4,11 @@ abstract class StringConstants {
       "http://api.themoviedb.org/3/movie/popular?api_key=" +
           apiKey +
           "&language=en-US&page=1";
+  static const String uriPathSearch =
+      "https://api.themoviedb.org/3/search/movie?api_key=" +
+          StringConstants.apiKey +
+          "&language=en-US&query=";
```

Comment:
Try to not use a constant inside another constant

Reply:
Agree, move the APIKEY outside that const.